gigantum/gigantum-client
# (#1560) fixes activity note markdown

## Pull Request for Issue #1560

**Describe changes introduced by this PR (Bugs fixed, features added, docs updated, etc.)**:
Standardizes markdown stylesheet to be imported and used throughout the application.

**Please check if the PR fulfills these requirements**

  - PR is set to merge to integration
  - PR title is descriptive, as this becomes the commit message on squash and merge.
  - Any related issues are added to the extended commit message during squash and merge.
  - Tests have been added (for bug fixes/features)
  - Docs have been added/updated (for bug fixes/features)

**QA Test Steps and Exit Criteria**:
Test readme and add a note with same markdown and make sure they match.
Make sure to use bullets, bold, italics, headers and quotes.
